### PR TITLE
Add tuftool to builder-base

### PR DESCRIPTION
**Pull request by abhay-krishna**
Installing tuftool in builder base so it's readily available during builds.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: builder-base/Dockerfile [hunk 1]
```diff
@@ -19,16 +19,11 @@ ARG GOPROXY
...
 
-COPY *-checksum *.sh /
-
-# attribution generation support
-COPY generate-attribution/package*.json \
-    generate-attribution/generate-attribution \
-    generate-attribution/generate-attribution-file.js \
-    generate-attribution/LICENSE-2.0.txt \ 
-    /opt/generate-attribution/
+COPY *-checksum *.sh generate-attribution /
```
[the remark this answers is not in the document]
Comment (abhay-krishna) [reply builder-base/Dockerfile:26]: I was wondering if we need the WORKDIR line at all...

[the remark this answers is not in the document]
Comment (abhay-krishna) [reply builder-base/Dockerfile:26]: yeah COPY, ADD and RUN create layers so definitely an issue

**Review by jaxesn — APPROVED** [review]

**Pull request merged**